src/api/models/attachment.ts

```diff
@@ -14,6 +14,7 @@ export interface Attachment {
     comment: string;
     mediaTypeDescription: string;
     fileId: string;
+    collectionName: string;
```

Comment:
Are you sure this extension always exists? Probably make sense to mark this property as optional?

Reply:
Will add as optional as not documented as always there.